nbs/02_match_class.ipynb

```diff
@@ -224,7 +224,9 @@
     "        self.graph: DiGraph = graph\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # Collections Feature
</pre><p>I'll address the discussion on which versions to use in github directly as a reply to your mail</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>